WordPress/Jetpack/Classes/ViewRelated/WordPress-to-Jetpack Migration/Common/Views/MigrationStepView.swift

```diff
@@ -71,13 +78,17 @@ class MigrationStepView: UIView {
 
     override func layoutSubviews() {
         super.layoutSubviews()
-        mainScrollView.contentInset.bottom = actionsView.frame.size.height + Constants.bottomMargin
+        let bottomInset = actionsView.frame.size.height - safeAreaInsets.bottom
+        mainScrollView.contentInset.bottom = bottomInset + Constants.bottomMargin
```

Comment:
Here, the idea is to inset the scrollView bottom by the Actions Sheet View height plus some margin that we define. But we have to take into account that the system already added insets to `scrollView` based on the parent view's safe area margins. The real `scrollView` content inset is the [`adjustedContentInset` property.](https://developer.apple.com/documentation/uikit/uiscrollview/2902259-adjustedcontentinset)